src/overlays/actors/ovl_En_Ba/z_en_ba.c

```diff
@@ -300,16 +301,22 @@ void EnBa_SwingAtPlayer(EnBa* this, PlayState* play) {
         }
         this->unk_2A8[13].x = this->unk_2A8[12].x;
         this->unk_2A8[13].y = this->unk_2A8[12].y;
+
+        //! @bug This code being located here gives multiple opportunities for the current action to change
+        //! before damage handling can be done.
+        //! By, for example, taking damage on the same frame the collider contacts Player, a different action
```

Comment:
```suggestion
        //! For example, by taking damage on the same frame the collider contacts Player, a different action
```
(I found the other way slighly awkward)